tests/acceptance/pageObjects/FilesPageElement/expirationDatePicker.js

```diff
@@ -98,7 +98,9 @@ module.exports = {
      */
     isExpiryDateDisabled: async function (pastDate) {
       let disabled = false
-
+      if (pastDate.getMonth() === 11) {
```

Comment:
`11` is "magic". At first I thought it is some hard-coded reference to November (the 11th month). Now I have read the PR and it seems to be the zero-based index of December (the 12th month).

Please add a comment. Or put this in a constant somewhere (with a comment) and use the constant here and at line 169.

Reply:
I changed the function, it should be easier to understand now.